### DISCUSSION TITLE
FSTAB to be expanded

### DISCUSSION THREAD

**Discussion by Reinemann67** [ltsp/ltsp#224] [Q&A]
I like to have access to a additional nfs folder. The folder is available and from another linux server I can access to that folder.
As described I added at the ltsp-server in /etc/ltsp/ltsp.conf 
< 
[common]
FSTAB_x="192.168.1.11:/share /share nfs auto,rw, 0 0"
>
After that I created a new image. 
ltsp image /

but it seems this line wound be used during the client boot sequenz.  
After the client boots nor entry in the client fstab is available neither the share has been mounted :-(

**Comment by alkisg** [comment]
After editing ltsp.conf, run `ltsp initrd`, not `ltsp image`.

Also, FSTAB isn't processed by the server, so better put it under [clients], not [common].

Finally, make sure that /share exists on the client, otherwise it won't get mounted. E.g. with

```shell
[clients]
POST_INIT_MKDIR_SHARE="mkdir -p /share"
FSTAB_MOUNT_SHARE="192.168.1.11:/share /share nfs auto,rw, 0 0"
```

**Comment by Reinemann67** [comment]
Many thanks, it is working now !
